File: aprendizado/exercicios/AndroidApp/app/src/main/java/com/example/androidapp/MainActivity.java
package com.example.androidapp;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import androidx.appcompat.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        Button bmiButton = findViewById(R.id.bmiButton);
        Button gradeCalculatorButton = findViewById(R.id.gradeCalculatorButton);
        Button developerInfoButton = findViewById(R.id.developerInfoButton);

        bmiButton.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(MainActivity.this, BmiActivity.class);
                startActivity(intent);
            }
        });

        gradeCalculatorButton.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(MainActivity.this, GradeCalculatorActivity.class);
                startActivity(intent);
            }
        });

        developerInfoButton.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(MainActivity.this, DeveloperInfoActivity.class);
                startActivity(intent);
            }
        });
    }
}

File: aprendizado/exercicios/MyAndroidApp/app/src/main/java/com/example/myandroidapp/MainActivity.kt
package com.example.myandroidapp

import android.content.Intent
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        btnBmiCalculator.setOnClickListener {
            val intent = Intent(this, BmiCalculatorActivity::class.java)
            startActivity(intent)
        }

        btnGradeCalculator.setOnClickListener {
            val intent = Intent(this, GradeCalculatorActivity::class.java)
            startActivity(intent)
        }

        btnAboutDeveloper.setOnClickListener {
            val intent = Intent(this, AboutDeveloperActivity::class.java)
            startActivity(intent)
        }
    }
}